build/codegen/codegen.sh

```diff
@@ -14,29 +14,58 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-GROUP_VERSIONS="ceph.rook.io:v1"
+##GROUP_VERSIONS="ceph.rook.io:v1"
+
+scriptdir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+basedir="$(cd "${scriptdir}/../.."  && pwd)"
+apisdir="${basedir}"/pkg/apis
```

Comment:
`apisdir` is particularly hard to read. Change to use snake case typical of bash scripts

```suggestion
base_dir="$(cd "${scriptdir}/../.."  && pwd)"
apis_dir="${basedir}"/pkg/apis
```